vcbuild.bat

```diff
@@ -14,7 +14,7 @@ if /i "%1"=="/?" goto help
 @rem Process arguments.
 set config=Release
 set target=Build
-set target_arch=x86
+set target_arch=x64
```

Comment:
Is it possible to detect if the OS is 64-bit like the Makefile does on the Unix platforms?